installer/src/Installer/Program.cs

```diff
@@ -209,9 +199,30 @@ private static FileVersionInfo GetDriverFileInfo(string zipContentsDirectory)
 		}
 	}
 
+	// Implemented as a CustomAction as using the WixShellExecTarget only ever launches a 32 bit process,
+	// then attempting to access any 64 bit programs will succumb to windows path rewriting and resolve to
+	// 32 bit equivalent program.
+	public class CustomActions
```

Comment:
I think this custom action should be implemented in a separate DLL, and included in the installer files. As is, the complete Installer program exe will be bundled into the installer just for this one custom action.

Reply:
Fair point, will extract and reference.

Reply:
@russcam - Changes made as requested.